src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.test.tsx

```diff
@@ -28,10 +28,23 @@ import ViewInApp from 'src/shared/components/view-in-app/ViewInApp';
 import { createGene } from 'tests/fixtures/entity-viewer/gene';
 import { createTranscript } from 'tests/fixtures/entity-viewer/transcript';
 
+jest.mock('@apollo/client', () => ({
+  gql: jest.fn(),
+  useQuery: () =>
+    jest.fn(() => ({
+      data: null,
+      loading: true
+    }))
```

Comment:
useQuery in this case would actually be the actual jest.fn rather that a function that returns jest.fn. Funnily enough, tests didn't care.

Reply:
✅